configuration-ghc-88.nix

```diff
@@ -0,0 +1,58 @@
+{ pkgs, inputs }:
+
+let
+  disabledPlugins = [
```

Comment:
Similarly, are theses plugins broken with ghc 8.8?

Reply:
It came as I copied the config file from newer versions, I don't know much how `disabledPlugins` work. I'm happy to iterate on this in a separate task

Reply:
It's fixed now